tests/test_invenio_communities.py

```diff
@@ -284,34 +284,39 @@ def assert_header_links(response, ref, page, size):
 
 def test_communities_rest_get_details(app, db, communities):
     """Test the OAI-PMH Sets creation."""
+    (comm1, comm2, comm3) = communities
     with app.test_client() as client:
-        response = client.get('/api/communities/comm1')
+        response = client.get('/api/communities/{}'.format(str(comm1.id)))
```

Comment:
> This wouldn't be an issue if this wasn't creating potentially thousands of row in the database as they are versioned.

This should not influence our architectural decision here. Space is cheap and community should not be renamed everyday.